atomic_reactor/plugins/pre_koji.py

```diff
@@ -66,7 +66,8 @@ def run(self):
             self.log.info('no target provided, skipping plugin')
             return
 
-        if self.workflow.builder.base_from_scratch and not self.workflow.builder.parent_images:
+        if (self.workflow.builder.dockerfile_images.base_from_scratch and
+                not self.workflow.builder.dockerfile_images.keys()):
```

Comment:
`.keys()` are not needed

Reply:
they are needed, because IF we will have just scratch base image
dockerfile_images WILL be defined BUT there won't be any pullable_parents that is why I am checking keys()

Reply:
Now I'm curious about that object protocol when `bool()` and `len()` are not tied with `keys()`

Reply:
nvm, you are right, keys() can be removed